webapp/components/Modal/DisableModal.vue

```diff
@@ -72,8 +72,8 @@ export default {
         this.$toast.success(this.$t('disable.success'))
         this.isOpen = false
         setTimeout(() => {
-          this.$emit('close')
-        }, 1000)
+          this.$emit('close'), location.reload()
```

Comment:
Please avoid two command on one line. Make it obvious:

```suggestion
          this.$emit('close')
          location.reload()
```